Source/Urho3D/Engine/Engine.cpp

```diff
@@ -767,13 +767,7 @@ void Engine::ApplyFrameLimit()
 
 #ifndef __EMSCRIPTEN__
     // Perform waiting loop if maximum FPS set
-#if !defined(IOS) && !defined(TVOS)
-    if (maxFps)
-#else
-    // If on iOS/tvOS and target framerate is 60 or above, just let the animation callback handle frame timing
-    // instead of waiting ourselves
-    if (maxFps < 60)
-#endif
+    if (maxFps && maxFps < 60)
```

Comment:
But if maxFps > 60, frame limiter will be disabled
And maxFps is 200 by default on Windows.

Reply:
rewrote